web/moduly/finance.php

```diff
@@ -16,18 +16,13 @@
 $a=$u->koncA();
 $uid=$u->id();
 
-if(!$zaplaceno)
-{
+if(!$zaplaceno) {
   $castka=-$fin->stav();
-  $pozde=-round($fin->stavPozde());
-  if(SLEVA_AKTIVNI)
-    $limit=datum3(SLEVA_DO);
+  $limit=datum3(GC_ODHLASENI);
```

Comment:
Tím, že jsi proměnnou $limit místo dynamicky dal fixně, tak jsi dost změnil chování kódu tady i níž. Tím ti tam můžou vzniknout nějaké always-true zbytečné podmínky a podobné haluze a bylo by potřeba to uklidit.

Reply:
Tady jsem změnil chování kódu dost výrazně, předělal jsem to jen je to trochu rozsáhlý.